src/data/int/modeq.lean

```diff
@@ -138,18 +138,18 @@ begin
   exact h.mul hd,
 end
 
-theorem of_modeq_mul_left (m : ℤ) (h : a ≡ b [ZMOD m * n]) : a ≡ b [ZMOD n] :=
+theorem of_mul_left (m : ℤ) (h : a ≡ b [ZMOD m * n]) : a ≡ b [ZMOD n] :=
 by rw [modeq_iff_dvd] at *; exact (dvd_mul_left n m).trans h
 
-theorem of_modeq_mul_right (m : ℤ) : a ≡ b [ZMOD n * m] → a ≡ b [ZMOD n] :=
-mul_comm m n ▸ of_modeq_mul_left _
+theorem of_mul_right (m : ℤ) : a ≡ b [ZMOD n * m] → a ≡ b [ZMOD n] :=
+mul_comm m n ▸ of_mul_left _
```

Comment:
I think this naming might be deliberate in order to refer to the argument within the `[]` rather than the other two arguments.

Reply:
I do not buy your explanation. When we remove `c *` from arguments of a relation, we call the lemma `cancel_mul_left_...`, not `of_mul_left`.

Reply:
I think there's still enough confusion that it would be helpful to have a docstring along the lines of:
```lean
/-- Cancel left multiplication in the modulus.

For cancelling a multiplication on both sides of the `≡`, see `nat.modeq.mul_left_cancel'`. -/
```